src/directives/index.js

```diff
@@ -1,7 +1,7 @@
 /**
  * This source file is part of the Swift.org open source project
  *
- * Copyright (c) 2021 Apple Inc. and the Swift project authors
+ * Copyright (c) 2021-2023 Apple Inc. and the Swift project authors
```

Comment:
I don't think we made changes to this file

Reply:
no clue how this was modified 🙃  fixed